odoo/addons/base/views/res_partner_views.xml

```diff
@@ -663,10 +664,14 @@
             <field name="model">res.partner.category</field>
             <field name="arch" type="xml">
                 <search string="Search Partner Category">
-                    <field name="name"/>
-                    <field name="display_name"/>
+                    <field name="name" string="Name"/>
+                    <field name="display_name" string="Category"/>
```

Comment:
Isn't it possible to use filter_domain instead of a (wrong) search on display_name ? Something like ``filter_domain=[('child_ids', 'ilike', self)]``